### DISCUSSION TITLE
For hoppscotch.io, where collections and workspaces are stored and what is their security?

### DISCUSSION THREAD

**Discussion by DaceKonn** [hoppscotch/hoppscotch#3020] [Q&A] [answered]
Just like in the title.

When I use hoppscotch.io and I sync my data, or share with team.
I couldn't find any information other than the collections are synced through cloud and local storage.

But can someone elaborate on where it stored exactly? What is the security?

I'm noticing that are some dangling discussions around this topic:
https://github.com/hoppscotch/hoppscotch/discussions/2956
https://github.com/hoppscotch/hoppscotch/discussions/2343

**Comment by tomyo** [comment]
Where/how is data stored when using the cloud version?  

I'm afraid to share my company secrets by using this. It's a shame that is not transparent.

**Comment by liyasthomas** [comment, marked as answer]
Hoppscotch mainly has 3 variants:

1. Hoppscotch Community Edition: [Self-host from GitHub →](https://github.com/hoppscotch/hoppscotch)

    * Install, administer, and maintain on your own.
    * Self-managed so you can deploy in your own data center or cloud capacity.

2. Hoppscotch Cloud: [Sign up on hoppscotch.io →](https://hoppscotch.io)

    * You don’t need to install anything to use Hoppscotch Cloud, simply sign up and you can start using it right away.
    * Fully managed by Hoppscotch using our cloud-hosted instances, means that your team can focus on shipping APIs instead of maintaining infrastructure.
    * Cloud exclusive features.

3. Hoppscotch Enterprise Edition (coming soon): [Join waiting list →](https://hoppscotch.io/beta)

    * Install, administer, and maintain on your own with our priority support.
    * Self-managed so you can deploy in your own data center or cloud capacity.
    * Enterprise-grade features and plugins like SSO authentication, usage insights, activity logs, etc.

Read more about Hoppscotch clients in our [Quick Start Documentation](https://docs.hoppscotch.io/documentation/getting-started/quick-start).

**Reply by lucacavallaro** [reply]
@liyasthomas is hoppscotch.io compliant with EU GDPR?
